app/authentication/resources.py

```diff
@@ -0,0 +1,40 @@
+from flask import session
+from flask_restful import Resource
+
+from app.users.models import User
+from app.lib.response_util import buildOkResponse, _returnUser
+
+
+class UserLogin(Resource):
+    """Class for logging in a user.
+    """
+    def get(self, userId):
+        user = User.query.get(userId)
+        session['id'] = user.id
+
+        return buildOkResponse(_returnUser(user))
+
+    def post(self, userId):
+        return self.get(userId)
+
+
+class UserLogOut(Resource):
+    """Class for logging out a user.
+    """
+    def get(self):
+        session.pop('id', None)
```

Comment:
Please add a doc string, but other than that, this method looks great! For some reason when I built it, I had an if statement to check for an `id` before popping. That was totally not needed, and this is way simpler.